.github/workflows/prettier-check.yml

```
@@ -0,0 +1,27 @@
+  pull_request:
+    branches: ['**']
+
+permissions:
+  contents: read
+
+jobs:
+  prettier:
+    name: Check formatting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)